### PR TITLE
#139456255 features to increase invites to eleven

### PR DESCRIPTION
**Pull request by eshaibu**

### CODE REVIEW

**Review by iampikuda — APPROVED** [review]

File: config/socket/socket.js [hunk 1]
```diff
@@ -142,6 +142,9 @@ module.exports = function (io) {
         }
       } else {
         // TODO: Send an error message back to this user saying the game has already started
+        console.log(socket.id,' is too late')
```
Comment (ooduye) [inline config/socket/socket.js]: Get rid of console log

File: public/css/style.css [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+  background-color: lightblue;
+}
+
+.search-result {
+  color: black;
+  padding: 2px 10px;
+  cursor: pointer;
+}
+
+/* older css*/
```
Comment (ooduye) [inline public/css/style.css]: Be more specific or get rid of it completely

File: public/js/controllers/game.js [hunk 3]
```diff
@@ -1,171 +1,209 @@
...
+      if (game.gameID && game.state === 'awaiting players') {
+        if (!$scope.isCustomGame() && $location.search().game) {
           // If the player didn't successfully enter the request room,
           // reset the URL so they don't think they're in the requested room.
-        $location.search({});
-      } else if ($scope.isCustomGame() && !$location.search().game) {
-          // Once the game ID is set, update the URL if this is a game with friends,
+          $location.search({});
+        } else if ($scope.isCustomGame() && !$location.search().game) {
+            // Once the game ID is set, update the URL if this is a game with friends,
```
Comment (ooduye) [inline public/js/controllers/game.js]: Fix comment indentation

File: public/js/controllers/game.js [hunk 4]
```diff
@@ -1,171 +1,209 @@
...
+      if ($scope.isCzar()) {
+        game.pickWinning(winningSet.card[0]);
+        $scope.winningCardPicked = true;
+      }
+    };
+  
+    $scope.winnerPicked = () => game.winningCard !== -1;
+  
+      //start game only if players not less than players limit
+      $scope.startGame = () => {
```
[the remark this answers is not in the document]
Comment (ooduye) [reply public/js/controllers/game.js]: Check this indentation flag

File: public/js/controllers/game.js [hunk 5]
```diff
@@ -1,171 +1,209 @@
...
+      return '#f9f9f9';
+    };
+  
+    $scope.pickWinning = (winningSet) => {
+      if ($scope.isCzar()) {
+        game.pickWinning(winningSet.card[0]);
+        $scope.winningCardPicked = true;
+      }
+    };
+
```
[the remark this answers is not in the document]
Comment (ooduye) [reply public/js/controllers/game.js]: Find out and if possible fix this flag

File: public/js/controllers/game.js [hunk 6]
```diff
@@ -1,171 +1,209 @@
...
+      $scope.pickedCards = [];
+    });
 
-    // In case player doesn't pick a card in time, show the table
-  $scope.$watch('game.state', () => {
-    if (game.state === 'waiting for czar to decide' && $scope.showTable === false) {
-      $scope.showTable = true;
-    }
-  });
+      // In case player doesn't pick a card in time, show the table
```
Comment (ooduye) [inline public/js/controllers/game.js]: fix indentation

**Pull request merged**